compiler/main/driver.cpp

```diff
@@ -1261,6 +1262,7 @@ static ArgumentDescription arg_desc[] = {
  {"gpu-block-size", ' ', "<block-size>", "Block size for GPU launches", "I", &fGPUBlockSize, "CHPL_GPU_BLOCK_SIZE", NULL},
  {"gpu-arch", ' ', "<cuda-architecture>", "CUDA architecture to use", "S16", &fGpuArch, "_CHPL_GPU_ARCH", setEnv},
  {"gpu-ptxas-enforce-optimization", ' ', NULL, "Modify generated .ptxas file to enable optimizations", "F", &fGpuPtxasEnforceOpt, NULL, NULL},
+ {"gpu-specialization", ' ', NULL, "Enable [disable] an optimization that clones functions known to executed on a GPU locale", "N", &fGpuSpecialization, "CHPL_GPU_SPECIALIZATION", NULL},
```

Comment:
> Maybe "Enable [disable] an optimization that clones functions potentially executed on a GPU locale"?

That sounds accurate but I think misses the point that after we rewrite things so that the cloned copy is only used when we're on a GPU locale, and this assumptions lets us optimize in ways we wouldn't be able to otherwise.

How about this:

"Enable [disable] an optimization that clones functions into copies assumed to run on a GPU locale."

If I was trying to be more precise I think I'd go with this:

"Enable [disable] an optimization that clones functions into copies assumed to run on a GPU locale and then perform GPU-locale specific optimizations on those copies"

But I think that's getting too wordy.